.travis.yml

```diff
@@ -1,16 +1,22 @@
 language: ruby
 sudo: false
 cache: bundler
+branches:
+  only:
+    - master
+
+bundler_args: --without development
+
 rvm:
-  - 2.0.0
-  - 2.1.1
-  - 2.2.2
-  - 2.3.0
+  - 2.0
+  - 2.1
+  - 2.2
+  - 2.3.1
   - ruby-head
-  - jruby
+  - jruby-1.7
```

Comment:
Remove this line and it should fix the build. JRuby 1.7 is very old and we probably don't need to support it anymore.